source/microbit/help.c

```diff
@@ -68,61 +69,110 @@ STATIC const mp_doc_t help_table_types[] = {
+    {&microbit_display_animate_obj, "Use animate(img, delay, stride, start=0, async=False, repeat=False) to animate\nimage 'img' with 'delay' milliseconds and 'stride' pixels offset between\nframes. Optional: 'start' offset from left hand side, 'async' to run in the\nbackground, 'repeat' to loop the animation.\n"},
+    {&microbit_display_get_pixel_obj, "Use get_pixel(x, y) to return the display's brightness at LED pixel (x,y).\nBrightness can be from 0 (LED is off) to 9 (maximum LED brightness).\n"},
+    {&microbit_display_set_pixel_obj, "Use set_pixel(x, y, b) to set the display at LED pixel (x,y) to brightness 'b'\nwhich can be set between 0 (off) to 9 (full brightness).\n"},
+    // Pins
+    {&microbit_p0_obj, "micro:bit's pin 0 on the gold edge connector.\n"},
+    {&microbit_p1_obj, "micro:bit's pin 1 on the gold edge connector.\n"},
+    {&microbit_p2_obj, "micro:bit's pin 2 on the gold edge connector.\n"},
+    {&microbit_p3_obj, "micro:bit's pin 3 on the gold edge connector.\n"},
+    {&microbit_p4_obj, "micro:bit's pin 4 on the gold edge connector.\n"},
+    {&microbit_p5_obj, "micro:bit's pin 5 on the gold edge connector.\n"},
+    {&microbit_p6_obj, "micro:bit's pin 6 on the gold edge connector.\n"},
+    {&microbit_p7_obj, "micro:bit's pin 7 on the gold edge connector.\n"},
+    {&microbit_p8_obj, "micro:bit's pin 8 on the gold edge connector.\n"},
+    {&microbit_p9_obj, "micro:bit's pin 9 on the gold edge connector.\n"},
+    {&microbit_p10_obj, "micro:bit's pin 10 on the gold edge connector.\n"},
+    {&microbit_p11_obj, "micro:bit's pin 11 on the gold edge connector.\n"},
+    {&microbit_p12_obj, "micro:bit's pin 12 on the gold edge connector.\n"},
+    {&microbit_p13_obj, "micro:bit's pin 13 on the gold edge connector.\n"},
+    {&microbit_p14_obj, "micro:bit's pin 14 on the gold edge connector.\n"},
+    {&microbit_p15_obj, "micro:bit's pin 15 on the gold edge connector.\n"},
+    {&microbit_p16_obj, "micro:bit's pin 16 on the gold edge connector.\n"},
+    {&microbit_p19_obj, "micro:bit's pin 19 on the gold edge connector.\n"},
+    {&microbit_p20_obj, "micro:bit's pin 20 on the gold edge connector.\n"},
+    {&microbit_pin_write_digital_obj, "micro:bit, write_digital(choice) to the pin. You have two 'choice' values,\n0 (lo) or 1 (hi).\n"},
+    {&microbit_pin_read_digital_obj, "micro:bit, read_digital() value from the pin as either 0 (lo) or 1 (hi).\n"},
+    {&microbit_pin_write_analog_obj, "micro:bit, write_analog(value) to the pin. You can use a whole number between\n0 and 255 as the value.\n"},
+    {&microbit_pin_read_analog_obj, "micro:bit, read_analog() value from the pin. Wow, analog has lots of values\n(0 - 65535). Digital has only 0 and 1.\n"},
+    {&microbit_pin_is_touched_obj, "If pin is_touched() on micro:bit, return True. If nothing is touching the pin,\nreturn False.\n"},
+    // I2C
+    {&microbit_i2c_obj, "Communicate with named devices connected to micro:bit using the I/O pins.\n"},
```

Comment:
``Communicate with one or more named devices connected to micro:bit. Each named device has an 'address', communicates using I2C, and connects to the I/O pins.``

@ntoll Perhaps ^^ The unique thing about I2C is that you can address multiple devices using only 2 pins.